components/generic/LinkGroup.vue

```diff
@@ -21,7 +21,17 @@
           :destination="link.url"
           :link-class="linkClass"
         >
-          {{ link.text }}
+          <template v-if="link.icon">
+            <span class="icon-background">
+              <i
```

Comment:
Is `i` the right element to use here?

Reply:
An `i` is commonly used for icons, a `span` would be appropriate too. I don't have a preference.